### PR TITLE
Add binary url for documents

### PR DESCRIPTION
**Pull request by jackmaloney**
### Change description ###

There is a bug in DM store which mean if you just send normal url it will sanitise the url and try to fetch the document. DM store uses http and port 443 which is wrong, and so there is an error thrown.
If you send both the normal url and binary url it won’t sanitise the document url so this PR should fix that issue

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/integrationTest/java/uk/gov/hmcts/reform/sscs/controllers/CcdCallbackControllerTest.java [hunk 1]
```diff
@@ -344,7 +344,7 @@ private ExceptionCaseData exceptionCaseData(Map<String, Object> caseData) {
             VALUE,
             ScannedRecord.builder().docScanDate("2018-10-10")
                 .documentControlNumber("11111")
-                .documentLink(DocumentLink.builder().documentUrl("http://www.bbc.com").build())
+                .documentLink(DocumentLink.builder().documentUrl("http://www.bbc.com").documentBinaryUrl("binaryurl.com").build())
```
Comment (nitinprabhu) [inline src/integrationTest/java/uk/gov/hmcts/reform/sscs/controllers/CcdCallbackControllerTest.java]: would be nice to use `http://www.bbc.com/binary` as it is the case always

File: src/integrationTest/resources/mappings/case-creation-request.json [hunk 2]
```diff
@@ -63,7 +63,8 @@
         "documentFileName" : "11111.pdf",
         "documentDateAdded" : "2018-10-10",
         "documentLink" : {
-          "document_url" : "http://www.bbc.com"
+          "document_url" : "http://www.bbc.com",
+          "document_binary_url" : "binaryurl.com"
```
Comment (nitinprabhu) [inline src/integrationTest/resources/mappings/case-creation-request.json]: would be nice to use `http://www.bbc.com/binary` as it is the case always

File: src/integrationTest/java/uk/gov/hmcts/reform/sscs/controllers/CcdCallbackControllerTest.java (hunk 1 above)
Comment (jackmaloney) [reply src/integrationTest/java/uk/gov/hmcts/reform/sscs/controllers/CcdCallbackControllerTest.java]: Updated

File: src/integrationTest/java/uk/gov/hmcts/reform/sscs/ApplicationTests.java [hunk 3]
```diff
@@ -1,19 +0,0 @@
...
-import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.test.context.junit4.SpringRunner;
-
-@RunWith(SpringRunner.class)
-@SpringBootTest
-public class ApplicationTests {
-
-    @Test
-    public void sample_test() {
-        assertThat(2).isGreaterThan(1);
```
Comment (timja) [inline src/integrationTest/java/uk/gov/hmcts/reform/sscs/ApplicationTests.java]: but how will i know for sure that 2 is greater than 1?

**Pull request merged**